apigateway/outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "api_url" {
+  value = "https://${aws_api_gateway_rest_api.rest_api.id}.execute-api.${var.region}.amazonaws.com/${var.environment}"
+}
+
+output "api_arn" {
```

Comment:
Is it worth calling this `api_execution_arn`? The `arn` and the `execution_arn` are different